index.html

```diff
@@ -14,8 +14,15 @@
     gtag('config', 'G-4BQJHL7RYV');
     </script>    
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="./mobile-menu-style.css">
+    <link rel="stylesheet" href="./popup-style.css">
     <title>Portfolio</title>
 </head>
 <body tabindex="0">
```

Comment:
- [ ] [OPTIONAL] I think this section will look better if you add more space between your name and the paragraph. You can check the Figma design for reference.
![image](https://user-images.githubusercontent.com/43698511/132774517-aa8fa981-75ef-4a04-9d04-9ba6eb72d8b7.png)